content/extending-chat-widget/custom-identity-provider/index.mdx

```diff
@@ -0,0 +1,180 @@
+---
+weight: 60
+category: "extending-chat-widget"
+subcategory: "custom-identity-provider"
+title: "Custom Identity Provider"
+tagline: "Pair your own customer identity with LiveChat"
+desc: "Cross-platform chats for your applications"
+versionGroup: "chat-widget"
+---
+
+# Introduction
+
+## What is it for?
+
+If you have a more advanced service, handling user sessions, you can pair LiveChat customer identity with your own. As a result, your customers will never lose chats they made with your support. What is more, if you also use LiveChat on other platfroms (e.g., in mobile applications), their conversations will sync, making the communication even more seamless and natural. Mind the fact that Customer Identity is not customer metadata like name or email. If you would like to pass those you are probably looking for [updating session variables](https://developers.labs.livechat.com/docs/feature/CW-2556-custom-identity-provider-docs/extending-chat-widget/javascript-api/#update-session-variables).
```

Comment:
```suggestion
**Customer Identity Provider** was created for those who build more advanced integrations that handle user sessions. It allows you to match customer identities in LiveChat with those stored in your service. As a result, your customers will never lose chats they had with your support. What is more, if you also use LiveChat on other platforms (for example, in mobile applications), customer conversations will sync, making the communication even more seamless and natural. Mind the fact that _customer identity_ is not customer metadata such as name or email. If you would like to pass one of those, you are probably looking for [updating session variables](https://developers.labs.livechat.com/docs/feature/CW-2556-custom-identity-provider-docs/extending-chat-widget/javascript-api/#update-session-variables).
```

Reply:
I also feel that at this point, we should briefly explain "customer identity" since we're saying that it's not email or name

Reply:
I managed to describe it like: " _Customer identity_ is an id thats assigned to each customer by our API. Basing on that we can match data and chat history with certain person using our services." but Im not sure where to put it. Probably it should be defined at very beginning, maybe even above this paragraph? WDYT?